hypothesis-python/tests/ghostwriter/test_ghostwriter.py

```diff
@@ -118,7 +118,9 @@ def annotated_any(x: Any):
     pass
 
 
-space_in_name = type("a name", (type,), {"__init__": lambda self: None})
+space_in_name = type(
+    "a name", (type,), {"__init__": lambda self: None, "method": lambda cls: cls}
```

Comment:
If this is intended as a classmethod or staticmethod, you'll need to manually apply the decorator.  (if not, might be worth adding as two additional test cases!)

Reply:
I can switch it back to what it was (generating a test for a class itself) then. We do have other tests to test if the class methods and static methods are added.